### PR TITLE
ASM-6439 Dependency:Fixed chasis config issue with RedundancyPolicy

### PR DESCRIPTION
**Pull request by arunrordell**
Changed status code and property value.

### CODE REVIEW

File: lib/puppet/type/chassis_settings.rb [hunk 1]
```diff
@@ -74,14 +74,14 @@
...
-        '1'
-      when "powersupply"
-        '2'
-      else
-        raise "Invalid redundancy policy"
+        when "none"
+          '0'
+        when "grid"
+          '1'
+        when "alertonly"
```
Comment (ripienaar) [inline lib/puppet/type/chassis_settings.rb]: are we not going to support `alertonly` anymore at all? Do we specifically not want to support it is that why you removed it?

A better commit message that describes the why of changes would be great and help those reviewing

Comment (arunrordell) [inline lib/puppet/type/chassis_settings.rb]: @ripienaar The value for redundancy policy is 'alertonly',based on that value I changed in puppet module and asm-deployer module. Initially deployment failed while validation at asm-deployer level. 
image

Comment (ripienaar) [inline lib/puppet/type/chassis_settings.rb]: sorry, I mean to ask are we not going to support `powersupply` as a value anymore at all, why remove it as a option?

sorry for the confusion

Comment (arunrordell) [inline lib/puppet/type/chassis_settings.rb]: yes I got that. Sorry I will change that and submit PR. Thank You for letting me know.

Comment (ripienaar) [inline lib/puppet/type/chassis_settings.rb]: same PR, just update the commit please - and fix the related one

**Comment by gavin-scott** [comment]
:+1: 

DO NOT MERGE -- HOLD FOR 8.2.1

**Comment by arunrordell** [comment]
@gavin-scott  Please review this PR.

**Comment by gavin-scott** [comment]
@shaorong-zhu  8.2 PR

**Comment by gavin-scott** [comment]
@shaorong-zhu can you create a jenkins build for this and get it into 8..2.1?

**Pull request merged**